### PR TITLE
Point to 0.1 archive of docs and site version fixes

### PR DESCRIPTION
**Pull request by ldemailly**
Point to 0.1 archive of docs and site version fixes:
site.data.istio.version was used where it should be hardcoded to 0.1
(like in “our first release was …”) and vice versa (like in the footer)

There isn’t really a navigation (as I don’t know how to do that) but at
least the docs/welcome page points to 0.1

### CODE REVIEW

File: _docs/index.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ type: markdown
 
 # Welcome
 
+Here you will find the latest ({{site.data.istio.version}}) documentation, follow this link for the [documentation for the latest stable release (0.1)](https://istio.io/v-0.1/docs/)
```
Comment (geeknoid) [inline _docs/index.md]: This shouldn't be the first thing at the top of the Welcome page. How about this:

- Change the Welcome header to be Welcome to Istio <version number>

- Add a new paragraph at the end of the page that points to the stable version

**Comment by ldemailly** [comment]
build istio.github.io/presubmit

File: _docs/index.md (hunk 1 above)
Comment (ldemailly) [inline _docs/index.md]: good idea. see new rev - how about now ? also any idea how/where to make a real nav bar - or is this enough?

**Comment by ldemailly** [comment]
build istio.github.io/presubmit

**Comment by ldemailly** [comment]
I am waiting for the redirects to be in place before merging this so we don't pollute the search with duplicate blogs/ faq/ home page etc

**Comment by ldemailly** [comment]
redirects are in place so pushing this

**Pull request merged**